hack/cmd_util.sh

```diff
@@ -174,7 +174,8 @@ function os::cmd::internal::expect_exit_code_run_grep() {
 	os::test::junit::declare_test_start
 
 	local name=$(os::cmd::internal::describe_call "${cmd}" "${cmd_eval_func}" "${grep_args}" "${test_eval_func}")
-	echo "Running ${name}..."
+    local preamble="Running ${name}..."
```

Comment:
I realize this is a little absurd since the describer already says it's `executing ...` but @deads2k wanted the `Running` and the ellipsis at the end. If we don't care about the output changing, I can remove this var and the extra text